Examples/SwiftExample/SwiftExample/AppDelegate.swift

```diff
@@ -32,7 +32,8 @@ extension ResponseSet {
     static func makeTopRepositories() -> ResponseSet {
         let generator = FileBasedResponseGenerator(contentType: ContentType.applicationJson, directory: "LocalServer")
 
-        var set = ResponseSet(requestName: "search", evaluation: .suffix("search/repositories"))
+        //var set = ResponseSet(requestName: "search", evaluation: .suffix("search/repositories"))
+        var set = ResponseSet(requestName: "search", evaluation: .contains("search/repositories"))
```

Comment:
This was helpful to see as a use-case, but I try not to leave code commented out in the code that ends up in master, so would you mind reverting this file to remove this change?

Reply:
Sure i will revert this change.